recipe/cross_compile_support.sh

```diff
@@ -78,6 +78,7 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
                     "libnpp_devel:libnpp"
                     "libnvjpeg_devel:libnvjpeg"
                     "cuda_compat:nvidia_driver"
+                    "cuda_profiler_api":""
```

Comment:
IIUC this package only exists in the RHEL RPMs for CUDA 11.8, but is absent on CUDA 11.2

So we may need to either add `cuda_profiler_api` conditionally to this list (only when CUDA 11.8 is used) or we may need to change the install logic below to ignore missing packages

Reply:
Resolved in commit: https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/286/commits/062bbd66d3cc46f5d3c60c989db355a26f2378cb